.github/ISSUE_TEMPLATE/workflows/linting.yml

```diff
@@ -0,0 +1,18 @@
+name: Linting
```

Comment:
Change it to prettier, same with the filename

Reply:
sure

Reply:
can you please label it under gssoc 23

Reply:
Once it is ready to merge I will label it...